src/recipe/recipe.dao.ts

```diff
@@ -93,13 +100,14 @@ export class RecipeDAO {
       throw new NotFoundException();
     }
 
-    const file = await this.fileService.findFileById(recipeFromDb.fileId);
-    const fileUrl = await this.s3Service.generatePresignedUrl(file.key);
-
-    return new RecipeResponse(recipeFromDb, fileUrl);
+    return recipeFromDb;
   }
 
-  async findAllRecipes(query: FindAllRecipesDTO): Promise<RecipeResponse[]> {
+  async findAllRecipes(query: FindAllRecipesDTO): Promise<
+    (Recipe & { preparing: RecipePreparationSteps[] } & {
+      ingredients: RecipeIngredients[];
+    })[]
```

Comment:
please consider moving those return types to some kind of interface or type, DRY